core/templates/dev/head/css/oppia.css

```diff
@@ -2162,9 +2164,8 @@ md-card.oppia-dashboard-tile {
 
 .oppia-dashboard-list-view-item {
   background: #fff;
-  margin: 20px auto 0;
-  padding: 10px 20px;
-  width: 100%;
+  margin: 20px 7.5px 0px 7.5px;
+  padding: 10px 12.5px;
```

Comment:
@Arunabh98 You actually decreased the padding (20px ->12.5px). And IMO it looks better when padding is kept the same (20px). The current one looks a bit more cramped up.